mod.ts

```diff
@@ -85,6 +85,10 @@ export interface BuildOptions {
   redirects?: Redirects;
   /** Package.json output. You may override dependencies and dev dependencies in here. */
   package: PackageJsonObject;
+  /** Package manager used to install dependencies and run npm scripts.
+   * @default npm
+   */
```

Comment:
Maybe add some docs here to mention that a full path can be provided for this option?